src/applications/travel-pay/tests/components/ClaimDetailsContent.unit.spec.jsx

```diff
@@ -250,4 +250,112 @@ describe('ClaimDetailsContent', () => {
       expect(screen.queryByText('Documents you submitted')).to.not.exist;
     });
   });
+
+  describe('Amounts', () => {
+    it('renders submitted amount and reimbursement amount when both are provided and > 0', () => {
+      const screen = renderWithStoreAndRouter(
+        <ClaimDetailsContent
+          {...claimDetailsProps}
+          totalCostRequested={123.45}
+          reimbursementAmount={67.89}
+        />,
+        {
+          initialState: getState(),
+        },
+      );
+
+      expect(screen.getByText('Submitted amount of $123.45')).to.exist;
+      expect(screen.getByText('Reimbursement amount of $67.89')).to.exist;
+    });
+
+    it('renders only submitted amount when reimbursement amount is 0', () => {
+      const screen = renderWithStoreAndRouter(
+        <ClaimDetailsContent
+          {...claimDetailsProps}
+          totalCostRequested={55.55}
+          reimbursementAmount={0}
+        />,
+        {
+          initialState: getState(),
+        },
+      );
+
+      expect(screen.getByText('Submitted amount of $55.55')).to.exist;
+      expect(screen.queryByText(/Reimbursement amount of/)).to.not.exist;
+    });
+
+    it('renders only reimbursement amount when submitted amount is 0', () => {
```

Comment:
I don't think the submitted amount could be 0. If it is possible, I don't know that the reimbursement amount would be more than the submitted amount.

Reply:
Yeah, that logically makes sense. Without knowing that was _never_ the case within the lifecycle of a claim, I was just being super protective here. Can pull it out if it's wasteful defensiveness.